Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -59,8 +60,11 @@ def http_request(self, url_suffix: str, requests_kwargs) -> Dict:
 
         if res.status_code in self.error_codes:
             raise DemistoException(f'{self.error_codes[res.status_code]}\t({res.content.decode()})')
-
-        return res.json()
+        try:
+            return res.json()
+        except json.decoder.JSONDecodeError as error:
+            demisto.error(str(res.content))
+            raise error
```

Comment:
why do you need this? unless you handle the error

Reply:
not handling the error but this will keep us from wasting time on logs.